src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -0,0 +1,327 @@
+/**
+ * Copyright (C) 2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */

```

Comment:
I realize that even the copyright messages have `*` at the beginning.
It means that someone changed what Xilinx lawyers decided...

How a lawyer can understand "You may * not" for example?